docs/docs/userguide/CONFIGURATION.md

```diff
@@ -94,9 +94,27 @@ bamboo_admin_email_address = "<email address>"
 
     Please refer to [Sensitive Data](#sensitive-data) section.
 
+!!!info "Restoring from existing dataset"
+    If the [`dataset_url` variable](#restoring-from-backup) is provided, the following properties are ignored and you will need to use user
```

Comment:
In fact we may say `above properties` here or have a direct refer:
```
 If the [`dataset_url` variable](#restoring-from-backup) is provided, the Bamboo system admin credentials are ignored ...
```

Reply:
I have updated the message to
```
!!!info "Restoring from existing dataset"
    If the [`dataset_url` variable](#restoring-from-backup) is provided, the _Bamboo System Admin Credentials_ properties are ignored.
    You will need to use user credentials from the dataset to log into the instance.
```